### ISSUE TITLE
Alpha value algorithm question

### ISSUE THREAD

**Issue by GuoFengYung** [mkkellogg/GaussianSplats3D#114]
Hello, I would like to ask you about the algorithm of alpha value.
https://github.com/mkkellogg/GaussianSplats3D/blob/main/src/SplatMesh.js#L230C1-L238C16

Here I see a different algorithm from Inria 
(https://github.com/graphdeco-inria/diff-gaussian-rasterization/blob/59f5f77e3ddbac3ed9db93ec2cfe99ed6c5d121d/cuda_rasterizer/forward.cu#L335C1-L345C14). 
I would like to ask you why.

**Comment by mkkellogg** [comment]
In actuality the two methods are almost the same, with some slight modifications on my end that filter out splat fragments with very low opacity. My code also has some optimizations that ultimately make it harder to connect back to the original code, and harder to understand what is actually happening. An explanation as to why they're basically equivalent would be really convoluted and hard to follow. I should really just try to simplify the code so it is easier to make the connection back to the original work. I will work on that for a future release :)

**Comment by GuoFengYung** [comment]
Thanks for your reply!
In addition, I would like to ask where the order is mainly written?
I would also like to ask how the order of sorting is determined.
So far I've seen it written here:
https://github.com/mkkellogg/GaussianSplats3D/blob/main/src/worker/sorter.cpp
Thank you again for your thoughtful reply!

**Comment by mkkellogg** [comment]
The sorting algorithm is a counting sort and the actual sorting begins at this line: https://github.com/mkkellogg/GaussianSplats3D/blob/b0b96454550a997c017a1b408186dae58efa4a27/src/worker/sorter.cpp#L142
All the code that comes before this line is concerned with computing the distances of each splat from the camera. You can see that different pathways are taken depending on whether an integer-based sort or a floating point sort is used, and whether or not dynamic mode is enabled.

**Comment by GuoFengYung** [comment]
Thank you for your detail reply><~
